microsoft-word/microsoft-word-quiz.md

```diff
@@ -130,14 +130,17 @@ Both switch the margin options from left/right to inside/outside. But only one "
 - [ ] Click File > Options > Recover Unsaved Documents.
 - [x] Click File > Open > Recent > Documents.
 - [ ] Click File > Recent > Recover Unsaved Documents.
-```
-solutions found in Office2019:
-File -> Open -> Recent -> Recover Unsaved Documents
-File -> Info -> Manage Document -> Recover Unsaved Documents
+
+<details><summary>Explanation</summary>
```

Comment:
I prefer not to use this `tags`,  remember we had an issue with this 
https://github.com/Ebazhanov/linkedin-skill-assessments-quizzes/issues/1086
so please revert it back or use some other solution